guides/common/modules/ref_dynamic_partition_example.adoc

```diff
@@ -0,0 +1,29 @@
+[[dynamic-partition-example]]
+= Dynamic Partition Example
+
+By default {Project} does not ship with dynamic partitioning. However, dynamic partitioning is possible.
```

Comment:
Traditionally, disk layout is decided on the server without knowing how many disks or how big they are. Typical kickstart partitioning is written in "create boot partition, create root parition, create /home for the rest of the disk".

Dynamic partitioning is executed by the installation program, therefore hardware (memory, dicks, size of them) information can be utilized to create kickstart configuration "on-the-fly". 

This is probalby worth mentioning, something in this regard.

Reply:
> Typical kickstart partitioning is written in "create boot partition, create root parition, create /home for the rest of the disk".

Help me understand: do you mean in that in this section of the kickstart template or in this sequence of actions?

Reply:
I mean, in normal kickstart:

```
zerombr
clearpart --all --initlabel
autopart <%= host_param('autopart_options') %>
```

This tells anaconda (the installer): erase whole disk, do autopartitioning, enlarge one partition to maximum size and proceed.

But dynamic layout is different, you can write your own rules on how to paritition disk according to runtime information from the node - eg. disk sizes, number of drives, vendor or manufacturer etc.

Reply:
OK hopefully my update contextualizes this appropriately.